src/utils.ts

```diff
@@ -50,7 +52,15 @@ export function parseHTTPHeadersQuotedKeyValueSet(
        * "      = The ending double quote
        * $      = The end of the string
        */
-      parsedValues[normalizedName] = value.replace(/^"(.+(?="$))"$/, '$1');
+      const strippedValue = value.replace(/^"(.+(?="$))"$/, '$1');
+
+      if (-1 === valuesToNormalize.indexOf(normalizedName)) {
```

Comment:
Using `valuesToNormalize.includes(normalizedName)` and swap the if..else instead will be more explicit or simpler:
```ts
parsedValues[normalizedName] = valuesToNormalize.includes(normalizedName) ? normalizedValue.toLowerCase() : normalizedValue;
```